### DISCUSSION TITLE
第5节课作业(9班)

### DISCUSSION THREAD

**Discussion by RangeKing** [InternLM/Tutorial#289] [作业]
LMDeploy 作业

**Comment by cookieswolf** [comment]
[05-LMDeploy 大模型量化部署实践](https://www.mubu.com/doc/3wAnX7GxKzO)

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135587651

**Comment by xiaomile** [comment]
[第四次作业](https://github.com/xiaomile/InternLM-homework/tree/main/%E4%BD%9C%E4%B8%9A4)

**Comment by ShelterWFF** [comment]
class5: http://t.csdnimg.cn/f1zoU

**Comment by wocclyl** [comment]
第5节课作业
https://m1ti1x9upsq.feishu.cn/wiki/XUPbwPycriVfSfkffNkciBDynnh?from=from_copylink

**Comment by haizeigh** [comment]
第五课 https://note.youdao.com/s/DKAr49DE

**Comment by haizeigh** [comment]
https://note.youdao.com/s/AiIwO6He
视频 https://www.bilibili.com/video/BV1iW4y1A77P/?vd_source=d6b5a09dda93b2c223da08c9b8241a1f

**Comment by wangxuejiao** [comment]
https://zhuanlan.zhihu.com/p/677779274

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135602703

**Comment by RangeKing** [comment]
https://rangeking.github.io/#/page/5.%20lmdeploy%20hands-on

**Comment by nini0919** [comment]
https://morning-gaura-eed.notion.site/6e6deb19583544168a08bfc723036554

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135761029?spm=1001.2014.3001.5501
